frontend/src/components/processes/ProcessDetailContent.js

```diff
@@ -94,31 +79,14 @@ const ProcessDetailContent = ({
               </Descriptions>
               <TrackingFieldsContent entity={process}/>
             </TabPane>
-            <TabPane tab="Properties" key="2" style={{marginTop:8} }>
-              { process?.children_properties?.length > 0 &&
-                  <>
-                  <Title level={3} style={{marginTop: '20px'}}>Properties</Title>
-                    <ProcessProperties
-                        propertyIDs={process.children_properties}
-                        protocolName={protocolsByID[process.protocol]?.name}
-                    />
-                  </>
-              }
-              {process?.children_processes?.map((id, i) => {
-                  const process = processesByID[id]
-                  return ( process &&
-                      <>
-                        <ProcessProperties
-                            propertyIDs={process.children_properties}
-                            protocolName={protocolsByID[process.protocol]?.name}
-                        />
-                      </>
-                  )
-                })
-              }
-              </TabPane>
-            </Tabs>
-        </PageContent>
+          <TabPane tab="Properties" key="2" style={{ marginTop: 8 }}>
+            <Title level={3} style={{ marginTop: '20px' }}>Process Properties</Title>
```

Comment:
Try using these 2 titles for each property tab 
Shared process properties => for process level properties
Sample process properties => for processMeasurement  properties